docs/Config.md

```diff
@@ -213,6 +216,21 @@ os:
   openCommand: 'open {{filename}}'
 ```
 
+### EditCommand
```

Comment:
```suggestion
### Configuring File Editing
```
Slightly more domain-specific title

Reply:
Made the change, along with some better English in that section.